src/script/interpreter.cpp

```diff
@@ -467,7 +467,10 @@ bool EvalScript(std::vector<std::vector<unsigned char> >& stack, const CScript&
                 opcode == OP_DIV ||
                 opcode == OP_MOD ||
                 opcode == OP_LSHIFT ||
-                opcode == OP_RSHIFT)
+                opcode == OP_RSHIFT ||
+                opcode == OP_VERIF ||
+                opcode == OP_VERNOTIF ||
+                (fExec && opcode == OP_VER))
```

Comment:
How does OP_VER differ from a non-existent opcode?

Reply:
It's actually defined and exists in documentation.

Reply:
I think that's the reason why OP_VER and its variants are evaluated as disabled opcode while `OP_RESERVED` is evaluated as reserved.
actually other disabled opcodes(which throw disabled err) are based on history.